docker/package/tezos_setup_wizard.py

```diff
@@ -208,7 +208,10 @@ def yes_or_no(prompt, default=None):
 
 
 def list_connected_ledgers():
-    output = get_proc_output("sudo -u tezos tezos-client list connected ledgers")
+    output = get_proc_output(
+        "sudo -u tezos TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES "
```

Comment:
`TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES` copypaste makes me sad:(
AFAIR, `subprocess.run` has `env` argument, so you should be able to modify the current `os.environ` with `TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER` and pass updated env dict to the `run`

Reply:
I know, it makes me sad too. The `env` argument is what I tried first, but the thing is that (for some reason, idk, the way `sudo` handles environments?) it doesn't work with `sudo -u tezos`.

Reply:
:cry:

Reply:
What do you think about moving `sudo -u tezos TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES` to the `get_proc_output`?

Reply:
oh, btw, there is `--preserve-env` option for `sudo`

Reply:
Oh, `--preserve-env` is pretty cool, thanks! On its own it would still require copy-paste though, just less of it.

Moving `get_proc_call` (and combining with `sudo -E`) is the option with the least changes, so I guess we'll do that. I hope it's OK for readability that that function isn't only used for calling `tezos-client`, but also e.g. for `systemctl show tezos-node-<network>.service`.

Reply:
I lied. There's going to be some amount of copy-paste either way, either additions or deletions, and introducing _two_ dedicated functions (the equivalents of `proc_call` for checking the return code and of `get_proc_output` for getting the output) seems like an overkill. So I just added `-E` flags to `sudo` where necessary.
Please re-review.